lib/src/view/analysis/analysis_screen.dart

```diff
@@ -796,6 +624,19 @@ class _BottomBar extends ConsumerWidget {
                         : Icons.area_chart,
                   ),
                 ),
+              Expanded(
+                child: BottomBarButton(
+                  label: context.l10n.openingExplorer,
+                  onTap: () => pushReplacementPlatformRoute(
```

Comment:
I had made it so that you could go to the analysis board from the opening explorer as well. So users could go back and forth multiple times which would clutter the navigation stack, and it would take many clicks to exit afterwards. On second thought though, I've now removed the possibility to go from the explorer to the analysis board, which gets rid of the problem.

Reply:
Yeah I think going from analysis to explorer is natural and you want to have support for back button.

If you open the explorer from the tools tab, you're likely to want only the explorer feature.